app.py

```diff
@@ -1,27 +1,39 @@
 """Example bot which execute python expression and returns a local execution environment which is enough to teach the concept"""
 
 from flask import Flask, request, json
```

Comment:
import should be in alphabetical order and we should import builtin import first and then third-party and then local import 

The import should look like
`
import io
import sys
import traceback

from flask import Flask, request, json
`